tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -774,3 +774,66 @@ Feature: Send a sharing invitations
       | Co Owner        | folder        | FolderToShare  |
       | Uploader        | folder        | FolderToShare  |
       | Manager         | folder        | FolderToShare  |
+
+
+  Scenario Outline: send sharing invitation to a deleted group with different roles
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+    And user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    And the administrator has deleted group "grp1"
+    When user "Alice" sends the following share invitation using the Graph API:
+      | resourceType    | <resource-type>   |
+      | resource        | <path>            |
+      | space           | Personal          |
+      | sharee          | grp1              |
+      | shareType       | group             |
+      | permissionsRole | <permissionsRole> |
```

Comment:
can you change please style like here: https://github.com/owncloud/ocis/pull/8139#discussion_r1447095514